src/checkTargetMatchToPR.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
```

Comment:
let's use regexp and semver package, also let's figure out if semver considers a change like: from 0.1.0 to 0.2.0 as major or minor (due to the 0 as the first number)

Reply:
using regex and semver pkg now. semver pks considers 0.1.0 to 0.2.0 as minor udpate.. however documantation here(https://semver.org/#spec-item-4) says, Major version zero (0.y.z) is for initial development. Anything MAY change at any time. i think its best if we treat Major version zero updates as major..